### DISCUSSION TITLE
Custom email template for GitHub notifications

### DISCUSSION THREAD

**Discussion by gauravchaudhari02** [community/community#49280] [Enterprise]
### Select Topic Area

Question

### Body

Hi All,
Generally, we receive notifications from GitHub about any activity performed on our subscribed repository such as push, pull request/ issue activities, mentions. but, the notification email has a default template defined/created by GitHub. is there any way, we can add a custom email template for such notifications?

My GitHub Instance - GitHub Enterprise Server 3.5.1

Thanks in advance. 😃

**Comment by james-ecd** [comment]
To customize the email template used for notifications in GitHub Enterprise Server 3.5.1, you can follow these general steps:

1. Log in to your GitHub Enterprise Server instance with administrative access.
2. Navigate to the instance settings by clicking on the gear icon in the top right corner of the screen and selecting "Management Console".
3. Select the "Email Settings" tab from the left sidebar.
4. Under the "Email Templates" section, click on the "Customize" button to edit the default email template.
5. Use the text editor to make any desired changes to the email template. You can use variables to insert dynamic content, such as the repository name, the user who triggered the notification, or the type of activity that triggered the notification.
6. Preview the customized email template by clicking on the "Preview" button.
7. Once you are satisfied with your changes, click "Save" to save the new email template.
